doc/source/tutorial/tutorial_presenting_larray_objects.ipynb

```diff
@@ -363,7 +363,7 @@
    "cell_type": "markdown",
```

Comment:
<p>elementsof ?</p>

 _Reply via <a href='https://app.reviewnb.com/larray-project/larray/pull/752/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='40'/>

Reply:
Hmmm, never mind... This seems to be a bug in ReviewNB :(

Reply:
Sounds like it is. Unfortunately.